src/main/webapp/electionlist.html

```diff
@@ -94,10 +96,16 @@ <h3>Elections coming up in {{state}}</h3>
             {{#each stateElections}}
             <li class="list-group-item election-item">
               <div class="election-quick-details">
-                <p>{{this.name}}</p>
+                <div id="state-election-name">
```

Comment:
I think we need to change this "id" too right?, same with "national-election-name"

Reply:
why would we have to change it? This is how I access the election name for the query URL without mixing up the two types of elections.

Reply:
Couldn't there be multiple different state elections? you wrapped this in {{#each stateElection}}
Each id="<value>" is supposed to be unique.

Reply:
oh, I see what you mean. I was under the impression that referring to the div with that id would only capture that particular election because it's within the Handlebars framework.

Reply:
Let me see if I can pass the name of the object to the function as a parameter instead.

Reply:
I recommend testing the JS where there are multiple state and national elections to double check that everything works correctly :)